### DISCUSSION TITLE
安装新版本后怎样卸载旧版本

### DISCUSSION THREAD

**Discussion by feilongzaitian919** [Klocman/Bulk-Crap-Uninstaller#484] [Q&A]
安装5.5后，怎样卸载5.4旧版本

**Comment by Klocman** [comment]
You can download old versions in github releases or on fosshub. Why do you want to downgrade to 5.4?

**Comment by NebulaTechs** [comment]
打开安装目录，运行unins000.exe即可卸载，再之后再安装。
